packages/sdk/src/types/Stackable.ts

```diff
@@ -0,0 +1,80 @@
+import { RewardRedemptionParams } from './Rewards'
+import { GiftRedemptionParams } from './Gift'
+import { OrdersCreateResponse } from './Orders'
+import { ApplicableToResultList } from './ApplicableTo'
+import { DiscountVouchersTypes, DiscountVouchersEffectTypes, DiscountUnitVouchersEffectTypes } from './DiscountVoucher'
+import { SimpleProduct, SimpleSku } from './Products'
+import { LoyaltyPointsTransfer } from './Loyalties'
+
+interface StackableOptionsBase {
```

Comment:
both booleans here are already deprecated, so I think there is no point in having it here

Reply:
Thanks, fixed in: 21bcfa6c2c77387d1f87f0e882d9c438bc8d20f9